torch/csrc/jit/codegen/cuda/lower_fusion_simplifier.cpp

```diff
@@ -91,6 +91,15 @@ class UnaryOpInserter : private kir::ExprMutator {
         gop, IrBuilder::create<UnaryOp>(container, UnaryOpType::Set, out, in));
   }
 
+  void handle(ViewDtypeOp* vop) final {
+    auto out = vop->out();
+    auto in = vop->in();
+    auto container = out->container();
+    registerReplace(
+        vop,
+        IrBuilder::create<UnaryOp>(container, UnaryOpType::EraseType, out, in));
```

Comment:
I'm uncertain if we want to replace this op before lowering. We might consider reverting this in some of the follow up work.